dist/tools/tapsetup/tapsetup

```diff
@@ -213,73 +234,103 @@ del_ipv6_routes() {
     done
 }
 
+delete_iface() {
+    IFACE="$1"
+    echo "deleting ${IFACE}"
+    case "${PLATFORM}" in
+        FreeBSD|OSX)
+            ifconfig "${IFACE}" destroy ;;
+        Linux)
+            ip link delete "${IFACE}" ;;
+        *) ;;
+    esac
+}
+
 delete_bridge() {
     echo "deleting ${BRNAME}"
```

Comment:
```suggestion
```

`delete_iface` already does the printing - you can drop this.

Reply:
Done